### PR TITLE
WOE 12 cards

### PR DESCRIPTION
**Pull request by Simisays**
Back for Seconds was a pain but I think I got it working

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/back_for_seconds.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Name:Back for Seconds
+ManaCost:2 B
+Types:Sorcery
+K:Bargain
+A:SP$ Pump | Origin$ Graveyard | TgtZone$ Graveyard | RememberTargets$ True | TargetMin$ 0 | TargetMax$ 2 | TgtPrompt$ Choose up to two target creature card in your graveyard | ValidTgts$ Creature.YouOwn | SubAbility$ DBCheat | SpellDescription$ Return up to two target creature cards from your graveyard to your hand. If this spell was bargained, you may put one of those cards with mana value 4 or less onto the battlefield instead of putting it into your hand.
+SVar:DBCheat:DB$ ChangeZone | ChangeType$ Card.IsRemembered+cmcLE4 | ChangeTypeDesc$ You may put one of those cards with mana value 4 or less onto the battlefield instead of putting it into your hand.  | Origin$ Graveyard | Destination$ Battlefield | Hidden$ True | Mandatory$ True | Condition$ Bargain | ChangeNum$ 1 | SubAbility$ DBChangeAll
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/back_for_seconds.txt:6]: Mandatory wrong?

The Bmto Battlefield is a May

File: forge-gui/res/cardsfolder/upcoming/archon_of_the_wild_rose.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Name:Archon of the Wild Rose
+ManaCost:2 W W
+Types:Creature Archon
+PT:4/4
+K:Flying
+S:Mode$ Continuous | Affected$ Creature.enchanted+YouCtrl+Other | SetPower$ 4 | SetToughness$ 4 | AddKeyword$ Flying | Description$ Other creatures you control that are enchanted by Auras you control have base power and toughness 4/4 and have flying.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/archon_of_the_wild_rose.txt:6]: incomplete affected

File: forge-gui/res/cardsfolder/upcoming/back_for_seconds.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Name:Back for Seconds
+ManaCost:2 B
+Types:Sorcery
+K:Bargain
+A:SP$ Pump | Origin$ Graveyard | TgtZone$ Graveyard | RememberTargets$ True | TargetMin$ 0 | TargetMax$ 2 | TgtPrompt$ Choose up to two target creature card in your graveyard | ValidTgts$ Creature.YouOwn | SubAbility$ DBCheat | SpellDescription$ Return up to two target creature cards from your graveyard to your hand. If this spell was bargained, you may put one of those cards with mana value 4 or less onto the battlefield instead of putting it into your hand.
+SVar:DBCheat:DB$ ChangeZone | ChangeType$ Card.IsRemembered+cmcLE4 | OptionalPrompt$ Do you want to put one of those cards with mana value 4 or less onto the battlefield instead of putting it into your hand ? | Optional$ True | Origin$ Graveyard | Destination$ Battlefield | Hidden$ True | Condition$ Bargain | ChangeNum$ 1 | SubAbility$ DBChangeAll
+SVar:DBChangeAll:DB$ ChangeZoneAll | ChangeType$ Card.IsRemembered | Origin$ Graveyard | Destination$ Hand | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRememberbed$ True
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/back_for_seconds.txt:8]: typo

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/back_for_seconds.txt:8]: better rewrite with Defined

Comment (Hanmac) [reply forge-gui/res/cardsfolder/upcoming/back_for_seconds.txt:8]: the DBCheat probably need a Forget remembered?

or is that already handled by timestamp checks?

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/back_for_seconds.txt:8]: Defined for ChangeZoneAll is for a defined player I think?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/back_for_seconds.txt]: just avoid Remembering

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/back_for_seconds.txt]: Problem now is that you can only choose one card (even if both creatures have cmc le4)

File: forge-gui/res/cardsfolder/upcoming/brave_the_wilds.txt [hunk 4]
```diff
@@ -0,0 +1,9 @@
+Name:Brave the Wilds
+ManaCost:G
+Types:Sorcery
+K:Bargain
+A:SP$ Animate | ValidTgts$ Land.YouCtrl | TgtPrompt$ Select target land you control | Power$ 3 | Toughness$ 3 | Types$ Creature,Elemental | Duration$ Permanent | SubAbility$ DBChangeZone | Condition$ Bargain | Keywords$ Haste | SpellDescription$ If this spell was bargained, target land you control becomes a 3/3 Elemental creature with haste that's still a land.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/brave_the_wilds.txt:5]: > Some instant and sorcery spells require additional targets if they're bargained. You ignore those targeting requirements if those spells aren't bargained, and you can't bargain those spells unless you can choose the appropriate targets.

File: forge-gui/res/cardsfolder/upcoming/back_for_seconds.txt (hunk 3 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/back_for_seconds.txt]: Try `Defined$ Targeted.cmcLE4`

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/back_for_seconds.txt]: hmm still doesn't let me choose.

File: forge-gui/res/cardsfolder/upcoming/brave_the_wilds.txt (hunk 4 above)
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/brave_the_wilds.txt:5]: So it needs a CheckSvar Count$Bargain or something like that ?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/brave_the_wilds.txt:5]: No you need to work with TargetMin
(there should be some with kicker like that)

File: forge-gui/res/cardsfolder/upcoming/back_for_seconds.txt (hunk 3 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/back_for_seconds.txt]: + `ChooseFromDefined$`

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/back_for_seconds.txt]: thanks working now !

**Review by tool4ever — APPROVED** [review]

**Pull request merged**